pkg/pdutil/pd.go

```diff
@@ -400,14 +434,18 @@ func (p *PdController) GetPDScheduleConfig(
 
 // UpdatePDScheduleConfig updates PD schedule config value associated with the key.
 func (p *PdController) UpdatePDScheduleConfig(
-	ctx context.Context, cfg map[string]interface{},
+	ctx context.Context, cfg map[string]interface{}, post pdHTTPRequest, prefixs ...string,
```

Comment:
Seems the parameter `post pdHTTPRequest` is only used for unit test. So I think a better way to handle this is adding a delegate method to expose the interface and add a inner implement to export the pdRequest for tests

Reply:
yes, it's only used for test. @kennytm what do you think?

Reply:
@3pointer expose another method for `cmd/debug.go`, which only accepts a `ctx` and no `cfg` or `post` or `prefixes`.